### PR TITLE
TI flash fixes

### PR DESCRIPTION
**Pull request by mwasilew**
meta-ti-bsp: add fixes and improvements to flash.sh

### CODE REVIEW

File: meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in [hunk 1]
```diff
@@ -104,11 +105,14 @@ boardType=
...
 # dev PID and VID
 devID=
+# non-interactive mode
+nonInteractive="n"
 while true; do
 	case "$1" in
 		--wic) wicFile="${2}"; shift 2 ;;
 		--board-type) boardType="${2}"; shift 2 ;;
 		--dev-id) devID="${2}"; shift 2 ;;
+		--yes) nonInteractive="y"; shift 1 ;;
```
Comment (igoropaniuk) [inline meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in:115]: minor: you can just use `shift` without specifying `1`

File: meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in [hunk 2]
```diff
@@ -196,13 +200,15 @@ if [ -z "${mmcPath}" ]; then
 	error_exit "Can't detect UMS exposed device!"
 fi
 
-read -p "Confirm flashing ${wicFile} to ${mmcPath} (y/N) >" -n 1 -r
+if [[ $nonInteractive != "y" ]]; then
+    read -p "Confirm flashing ${wicFile} to ${mmcPath} (y/N) >" -n 1 -r
+fi
 echo ""
-if [[ $REPLY =~ ^[Yy]$ ]]; then
+if [[ $REPLY =~ ^[Yy]$ ]] || [[ $nonInteractive = "y" ]]; then
```
Comment (igoropaniuk) [inline meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in:207]: IMO vice-versa better:
`if [[ $nonInteractive = "y" ]] || [[ $REPLY =~ ^[Yy]$ ]];` 

as we can just skip checking `$REPLY` if `$nonInteractive = "y"` is true

File: meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in (hunk 1 above)
Comment (mwasilew) [reply meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in]: Fixed.

File: meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in (hunk 2 above)
Comment (mwasilew) [reply meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in:227]: Fixed.

Comment (igoropaniuk) [reply meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in:227]: hm, strange, I still see the old change

Comment (mwasilew) [reply meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in]: My mistake. I added it in one commit and reverted in the next one. Fixed now.

**Review by igoropaniuk — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**